File: VB/NewProject/Form1.vb
Imports System
Imports DevExpress.XtraEditors
Imports System.Data
Imports DevExpress.XtraCharts

Namespace NewProject
    Partial Public Class Form1
        Inherits XtraForm

        #Region "Properties"
        Public Property CurrentViewType() As ViewType
        #End Region
        Private chartHelper As ChartZoomScrollHelper
        Public Sub New()
            InitializeComponent()

            UpdateDataSource()
            InitializeChartControl()
        End Sub


        Private Function CreateData(ByVal rows As Integer) As DataTable
            Dim _r As New Random()

            Dim dt As New DataTable()
            dt.Columns.Add("Arg", GetType(Integer))
            dt.Columns.Add("Value", GetType(Integer))
            For i As Integer = 0 To rows - 1
                dt.Rows.Add(i, _r.Next(100))
            Next i
            Return dt
        End Function


        Private Sub InitializeChartControl()
            CurrentViewType = ViewType.Area
            chartControl1.Series.Clear()
            Dim lastSeriesIndex As Integer = chartControl1.Series.Add("MainSeries", CurrentViewType)

            chartControl1.Series(lastSeriesIndex).ValueDataMembers.AddRange(New String() { "Value" })
            chartControl1.Series(lastSeriesIndex).ArgumentDataMember = "Arg"
            CType(chartControl1.Diagram, XYDiagram).AxisY.VisualRange.Auto = False

            chartHelper = New ChartZoomScrollHelper(chartControl1)
        End Sub

        Private Sub UpdateDataSource()
            chartControl1.DataSource = CreateData(10)
        End Sub

    End Class
End Namespace

File: CS/NewProject/Form1.cs
using System;
using DevExpress.XtraEditors;
using System.Data;
using DevExpress.XtraCharts;

namespace NewProject
{
    public partial class Form1 : XtraForm
    {
        #region Properties
        public ViewType CurrentViewType { get; set; }
        #endregion
        ChartZoomScrollHelper chartHelper;
        public Form1()
        {
            InitializeComponent();

            UpdateDataSource();
            InitializeChartControl();
        }


        DataTable CreateData(int rows)
        {
            Random _r = new Random();

            DataTable dt = new DataTable();
            dt.Columns.Add("Arg", typeof(int));
            dt.Columns.Add("Value", typeof(int));
            for (int i = 0; i < rows; i++)
                dt.Rows.Add(i, _r.Next(100));
            return dt;
        }


        private void InitializeChartControl()
        {
            CurrentViewType = ViewType.Area;
            chartControl1.Series.Clear();
            int lastSeriesIndex = chartControl1.Series.Add("MainSeries", CurrentViewType);

            chartControl1.Series[lastSeriesIndex].ValueDataMembers.AddRange(new string[] { "Value" });
            chartControl1.Series[lastSeriesIndex].ArgumentDataMember = "Arg";
            ((XYDiagram)chartControl1.Diagram).AxisY.VisualRange.Auto = false;

            chartHelper = new ChartZoomScrollHelper(chartControl1);
        }

        private void UpdateDataSource()
        {
            chartControl1.DataSource = CreateData(10);
        }

    }
}
